core/services/pipeline/task.bridge.go

```diff
@@ -125,24 +159,29 @@ func (t *BridgeTask) Run(ctx context.Context, lggr logger.Logger, vars Vars, inp
 	var cachedResponse bool
 	responseBytes, statusCode, headers, elapsed, err := makeHTTPRequest(requestCtx, lggr, "POST", URLParam(url), []string{}, requestData, t.httpClient, t.config.DefaultHTTPLimit())
 	if err != nil {
+		promBridgeErrors.WithLabelValues(t.Name).Inc()
 		if cacheTTL == 0 {
 			return Result{Error: err}, RunInfo{IsRetryable: isRetryableHTTPError(statusCode, err)}
 		}
 
 		var cacheErr error
 		responseBytes, cacheErr = t.orm.GetCachedResponse(t.dotID, t.specId, cacheDuration)
 		if cacheErr != nil {
+			promBridgeCacheErrors.WithLabelValues(t.Name).Inc()
 			lggr.Errorw("Bridge task: cache fallback failed",
 				"err", cacheErr.Error(),
 				"url", url.String(),
 			)
 			return Result{Error: err}, RunInfo{IsRetryable: isRetryableHTTPError(statusCode, err)}
 		}
+		promBridgeCacheHits.WithLabelValues(t.Name).Inc()
 		lggr.Debugw("Bridge task: request failed, falling back to cache",
 			"response", string(responseBytes),
 			"url", url.String(),
 		)
 		cachedResponse = true
+	} else {
+		promBridgeLatency.WithLabelValues(t.Name).Set(elapsed.Seconds())
```

Comment:
Would it be worthwhile to keep some higher precision around here? maybe millis?

Reply:
This already gives high precision - duration.Seconds() returns float64 https://pkg.go.dev/time#Duration.Seconds

Reply:
Note that prometheus recommends using standard units - https://prometheus.io/docs/practices/naming/